README.md

```diff
@@ -395,6 +395,72 @@ The response will contain the following fields:
+
+### EhrStatusRequest
+
+| Field name               | Description                                                                                                     | Data type     | Possible values                                                                               | nullable |
+| ------------------------ | --------------------------------------------------------------------------------------------------------------- | ------------- | --------------------------------------------------------------------------------------------- | -------- |
+| initialRequestTimestamp  | The date and time of the original request.                                                                      | ISO-8601      |                                                                                               | False    |
+| actionCompletedTimestamp | The date and time of when the transfer completed.                                                               | ISO-8601      |                                                                                               | False    |
+| nhsNumber                |                                                                                                                 | string        |                                                                                               | False    |
+| conversationId           |                                                                                                                 | string        |                                                                                               | False    |
+| fromAsid                 |                                                                                                                 | string        |                                                                                               | False    |
+| toAsid                   |                                                                                                                 | string        |                                                                                               | False    |
+| fromOdsCode              |                                                                                                                 | string        |                                                                                               | False    |
+| toOdsCode                |                                                                                                                 | string        |                                                                                               | False    |
+| migrationStatus          | The current state of the transfer, a status of COMPLETE_WITH_ISSUES is given if placeholder documents were sent | string / enum | COMPLETE <br/> COMPLETE_WITH_ISSUES <br/> FAILED_NME <br/> FAILED_INCUMBENT <br/> IN_PROGRESS | False    |
```

Comment:
Can this field be in-progress,  line 401 says they are excluded?

Reply:
I guess it's part of the business requirements in-progress transfers are excluded. I would have thought it would be useful to see them.

Reply:
Assuming that all transfers eventually finish, I can see the logic behind only including "not in-progress" transfers, as that way the values displayed are final.

Whether that logic is right or wrong, I can't say 😄